test_unstructured/partition/test_api.py

```diff
@@ -214,13 +185,17 @@ def test_partition_via_api_valid_request_data_kwargs():
     assert isinstance(elements, list)
 
 
+@pytest.mark.skipif(skip_outside_ci, reason="Skipping test run outside of CI")
+@pytest.mark.skipif(skip_not_on_main, reason="Skipping test run outside of main branch")
```

Comment:
Is there a reason you're skipping this test outside of `main`?

Reply:
I saw that this decorator was used on all the tests that required an api key and added it to this one as well. It did strike me as a potential problem as well, but this is an integration test that makes a real request to the API, so I assumed that was why it was skipped on main.

Also, I met with @scanny yesterday to look over this a little and he pointed out that the mock response at the top of the file is more a fake response and we should have asserts against an actual mock in the tests, so I'm going to work on updating those before merging this, so I'm happy to make changes to these tests as well while I'm at it if we want to do something other than skipping on main.

Reply:
Gotcha, if this is a test that requires an API that checks out. Just wanted to make sure we weren't disabling a test we wanted to run on branches. Thanks!